README.md

```diff
@@ -323,43 +396,92 @@ An Oracle model definition consists of the following properties:
           "nullable":"N"
         }
       },
-      "available":{
-        "type":"Number",
-        "required":false,
-        "length":22,
+    }
+  }
+```
+
+</details>
+
+**Notice**: the Oracle database default type is UPPERCASE. If the oracle settings are not specified in the model, for example:
+
+```ts
+export class Demo extends Entity {
+  @property({
+    type: 'number',
+    required: true,
+    id: true,
+  })
+  id: number;
+```
+
+the connector would look for a table named `DEMO` under the default schema in the database and also tend to map the id property to a column named `ID` in that table. This might cause errors if the default table/colum name doesn't exist. Please do specify the settings if needed.
+
+### Configure a custom table/column name
+
+On the other hand, such settings would also allow you to have different names for models and tables. Take the `User` model as an example, we can map the `User` model to the table `MYUSER` and map the `id` property to column `MY_ID`as long as they are specified correctly:
+
+```ts
+@model({
+  settings: {
+    oracle: {
+      schema: 'XE',
+      table: 'MYUSER' // customized name
+    }
+  }
+})
+export class User extends Entity {
+  @property({
+    type: 'number',
+    required: true,
+    id: true,
+    oracle: {
+      columnName: 'MYID' // customized name
+    },
+  })
+  id: number;
+  ...
```

Comment:
I think we try to make the code snippet as valid as possible. So I'm suggesting to add the comment for `//...` and closing bracket for the class.
```suggestion
  //...
  }
```